packages/features/bookings/lib/handleSeats/create/createNewSeat.ts

```diff
@@ -59,11 +59,13 @@ const createNewSeat = async (
   const videoCallReference = seatedBooking.references.find((reference) => reference.type.includes("_video"));
 
   if (videoCallReference) {
+    const isDailyVideo = videoCallReference.type === "daily_video";
```

Comment:
Let's not hardcode daily_video string here. I believe there are existing constant or something to know if it is daily video